content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -0,0 +1,80 @@
+===================================================
+Use product variants on quotations and sales orders
+===================================================
+
+Product variants can be added to quotations and sales orders using the Product Configurator,
+Variant Grid Entry, or both.
+
+Configuration
+=============
+
+To enable product variants on quotations and sales orders, head to the :guilabel:`Sales` app, go to
+:menuselection:`Configuration --> Settings`, and activate one or both of the
+:guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings.
+
+.. image:: orders_and_variants/activating-entry-type-settings.png
+   :align: center
+   :alt: Activating entry type settings
+
+Next, configure the product form. Head over to :menuselection:`Sales --> Products --> Products` and
+select any product (or click :guilabel:`Create` to create a new one). On the product form, click
+:guilabel:`Edit`, then click on the :guilabel:`Variants` tab. Under
+:guilabel:`Sales Variant Selection`, choose either :guilabel:`Product Configurator` or
+:guilabel:`Order Grid Entry`. This determines which method is used to add this product to
+quotations or sales orders.
+
+.. image:: orders_and_variants/sales-variant-selection.png
+   :align: center
+   :alt: Sales variant selection
+
+.. note::
+   The product must have at least two variants (one attribute with two values) created in order for
+   the :guilabel:`Sales Variant Selection` option to appear. Click :guilabel:`Add a line` under
+   :guilabel:`Attribute` to add attributes and values to this product and create product variants.
+
+Using the Product Configurator
+==============================
+
+The :guilabel:`Product Configurator` appears on a quotation or sales order when products are added
+that have both variants configured and the :guilabel:`Product Configurator` option selected.
+
+The :guilabel:`Product Configurator` lets salespeople choose exactly which product variant to add
+to the quotation or sales order using a format similar to online shopping. Color options display
+any HTML color codes set up in the :guilabel:`Color` attribute. Features with an additional cost
+will display the additional cost next to their name.
+
+.. image:: orders_and_variants/configurator-price-extras.png
+   :align: center
+   :alt: Product Configurator and Price Extras
```

Comment:
@mivu-odoo @samueljlieber Let's also please update this image to have a purple box instead of a red box :)
![configurator-price-extras](https://user-images.githubusercontent.com/101904966/187537661-8f459165-2947-4739-a11a-0185c63593ac.png)